Features/foutafhandeling.feature

```diff
@@ -45,207 +45,633 @@ Bij een fout op een parameter krijgt in "invalid-params" attribuut "code" een va
+    | name       | code   | reason   |
+    | <paraeter> | <code> | <reason> |
+
+    Voorbeelden:
+    | path     | query string                                                                                                     | parameter  | code    | reason                                         |
+    | adressen | ?postcode=1234AA&huisnummer=999999&huisletter=AA                                                                 | huisnummer | maximum | Waarde is hoger dan maximum {maximum}.         |
+    | adressen | ?postcode=1234AA&huisnummer=999999&huisletter=AA                                                                 | huisletter | pattern | Waarde voldoet niet aan patroon ^[a-zA-Z]{1}$. |
+    | /panden  | ?point=98095.02,438495.09&statusPand=Pand%20in%20gebruik&geconstateerd=true&bouwjaar[min]=-1&bouwjaar[max]=10000 | bouwjaar   | minimum | Waarde is lager dan minimum {minimum}.         |
+    | /panden  | ?point=98095.02,438495.09&statusPand=Pand%20in%20gebruik&geconstateerd=true&bouwjaar[min]=-1&bouwjaar[max]=10000 | bouwjaar   | maximum | Waarde is hoger dan maximum {maximum}.         |
+
+Rule: een verplichte parameter moet zijn opgegeven met een waarde
+
+  Scenario: Verplichte parameter is niet opgegeven
+    Als bij een endpoint met verplichte parameters wordt aangeroepen zonder een parameter
+    Dan bevat de response ten minste de volgende velden
+    | naam   | waarde                                          |
+    | title  | Ten minste één parameter moet worden opgegeven. |
+    | status | 400                                             |
+    | code   | paramsRequired                                  |
+    En bevat de response geen veld "invalidParams"
+
+Rule: minimaal één zoekparameter(combinatie) moet zijn opgegeven bij een collectieaanroep
 
   Scenario: geen enkele zoekparameter opgegeven in zoekvraag
-    Als resources worden gezocht zonder parameters
-    Dan is de http status code van het antwoord 400
-    En is in het antwoord title=Minimale combinatie van parameters moet worden opgegeven.
-    En is in het antwoord status=400
-    En eindigt attribuut instance met /panden
-    En is in het antwoord code=paramsRequired
-    En is in het antwoord title=Ten minste één parameter moet worden opgegeven
-    En komt attribuut invalid-params niet voor in het antwoord
-
-  Scenario: meerdere fouten in parameters
-    Gegeven dat pand 9914100010921152 bestaat
-    Als pand 9914100010921152 wordt gezocht met onbekend=a&fout=b
-    Dan is de http status code van het antwoord 400
-    En is in het antwoord title=Een of meerdere parameters zijn niet correct.
-    En is in het antwoord status=400
-    En eindigt attribuut instance met panden/9914100010921152?onbekend=a&fout=b
-    En bevat invalid-params exact 2 voorkomen(s)
-    En is er een invalid-params met name=onbekend
-    En is er een invalid-params met name=fout
+    Als <path> wordt aangeroepen zonder zoekparameters
```

Comment:
Met die quickfix voor Pink van laatste geldt dit specifieke scenario niet meer. Hier moeten we nog wat mee. @NathalieVos ter info.